bcgov/HOUS-permit-portal
# [bphh-144] Implement ability to add reference number to a submitted permit application by reviewer and review manager

## Description
 Implement ability to add reference number to a submitted permit application by reviewer and review manager



## What type of PR is this? (check all applicable)

- [X ] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-144


## Steps to QA
- Ensure there is a permit application submitted by a submitter
- Login as a review manager or reviewer and navigation to the permit application inbox
- Locate the submitted permit application and select to view it
- You should be able to locate the reference number edit in the header and add text to it
- Ensure text persists after saving

<img width="1728" alt="Screenshot 2024-03-14 at 12 41 50 PM" src="https://github.com/bcgov/HOUS-permit-portal/assets/32022335/6f11a116-72fd-40fe-9ae4-d8704bf27057">
